lib/puppetserver/ca/action/prune.rb

```diff
@@ -80,7 +80,12 @@ def prune_CRLs(crl_list)
 
         def update_pruned_CRL(crl_list, pkey)
           crl_list.each do |crl|
-            crl.version=(crl.version + 1)
+            extension_key = crl.extensions.select { |ext| ext.oid != "crlNumber" }
+            extension_number = crl.extensions.select { |ext| ext.oid == "crlNumber" }
```

Comment:
I'd write this as `number_extension` or `number_ext` instead of `extension_number`, I'd also call the other array, `other_extensions` since the kind and quantity of other extensions isn't important to this function.